aptos-move/framework/aptos-framework/sources/stake.move

```diff
@@ -429,29 +319,63 @@ module aptos_framework::stake {
         move_to(account, AptosCoinCapabilities { mint_cap })
     }
 
+    /// Initialize the validator account and give ownership to the signing account
+    /// except it leaves the ValidatorConfig to be set by another entity.
+    /// Note: this triggers setting the operator and owner, set it to the account's address
+    /// to set later.
+    public entry fun initialize_owner_only(account: &signer, operator: address, voter: address) acquires OwnerCapability, StakePool {
+        initialize_owner(account, false);
+
+        move_to(account, ValidatorConfig {
+            consensus_pubkey: vector::empty(),
+            network_addresses: vector::empty(),
+            fullnode_addresses: vector::empty(),
+            validator_index: 0,
+        });
+
+        let account_address = signer::address_of(account);
+        if (account_address != operator) {
```

Comment:
I think we can move this check (and the similar one below) into set_operator so that it becomes a no-op (no event emitted as well) if the address is the same.

Reply:
sure